tsconfig.json

```diff
@@ -3,6 +3,8 @@
   "compilerOptions": {
     "baseUrl": "./",
     "outDir": "./dist/out-tsc",
+    "resolveJsonModule": true,
+    "allowSyntheticDefaultImports": true,
```

Comment:
If you change to the `HttpClient` approach, please undo these changes.

Reply:
Done